agent/bench-scripts/pbench-fio

```diff
@@ -426,14 +426,16 @@ function fio_install() {
 # install python-pandas on the controller: fiologparser_hist.py needs it
 function pandas_install() {
 	if [ "$postprocess_only" = "n" ]; then
-		if check_install_rpm python-pandas; then
+	    if check_install_rpm python2-pandas; then
+			debug_log "[$script_name]python2-pandas is installed"
+		elif check_install_rpm python-pandas; then
```

Comment:
I think with `elif` you want to indent it to the same level as the associated `if`.